ui/app/components/console/command-input.js

```diff
@@ -17,16 +17,17 @@ export default Component.extend({
   actions: {
     handleKeyUp(event) {
       const keyCode = event.keyCode;
+      const val = event.target.value;
```

Comment:
I changed this to make it more consistent with the rest of the UI code so the implementation matches.